lncfg/address.go

```diff
@@ -18,6 +18,8 @@ var (
 	loopBackAddrs = []string{"localhost", "127.0.0.1", "[::1]"}
 )
 
+const allInterfacesIP = "0.0.0.0"
```

Comment:
This wouldn't handle the IPv6 version of this: `::`

Reply:
Also thinking about this more, we could remove the `loopBackAddrs` field above, and instead use: [`IsLoopback`](https://pkg.go.dev/net#IP.IsLoopback)

Reply:
ah, looks like the (net.IP).`IsLoopback` method does not compare agains the "localhost" string...